src/order/hom/basic.lean

```diff
@@ -796,8 +820,7 @@ variables [partial_order α] [partial_order β] [partial_order γ]
 @[simps apply]
 def with_top_congr (e : α ≃o β) : with_top α ≃o with_top β :=
```

Comment:
I feel like this should be called `order_iso.with_top`.

Reply:
I don't want to change API in the same PR. Once we merge this, feel free to open a small PR that renames this `order_iso`.